pkg/api/pluginproxy/pluginproxy.go

```diff
@@ -22,10 +22,10 @@ type templateData struct {
 
 // NewApiPluginProxy create a plugin proxy
 func NewApiPluginProxy(ctx *models.ReqContext, proxyPath string, route *plugins.Route,
-	appID string, cfg *setting.Cfg, store pluginsettings.Service, secretsService secrets.Service) *httputil.ReverseProxy {
+	appID string, cfg *setting.Cfg, pluginSettingsService pluginsettings.Store, secretsService secrets.Service) *httputil.ReverseProxy {
```

Comment:
Now it's switched around, argument named `pluginSettingsService` and interface `pluginsettings.Store`. Before it was argument named `store` and interface `pluginsettings.Service`. 

Sorry, not sure here, but feels more appropriate to keep Service interface and Service struct, e.g.
- pkg/services/pluginsettings/Service interface
- pkg/services/pluginsettings/service/Service struct

If we want to have a different interface for plugin management, a store?, we can add an internal one to plugin management. 

WDYT?